content/python/concepts/math-module/terms/math-radians/math-radians.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: 'math.radians()'
+Description: 'Returns the radian equivalent of an input angle given in degress.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Geometry'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **math.radians()** function takes in an angle measured in degrees and returns its radian equivalent.
```

Comment:
```suggestion
The **`math.radians()`** function takes an angle measured in degrees and returns its radian equivalent.
```

Reply:
the term should be represented in backticks (` `)

Reply:
fixed